.github/workflows/kuksa-client.yml

```diff
@@ -82,15 +82,16 @@ jobs:
           ${{ steps.meta.outputs.tags }}
         labels: ${{ steps.meta.outputs.labels }}
 
+    # Push currently disabled as it results in 413 Request Entity Too Large
     - name: Build ephemereal kuksa command line client docker and push to ttl.sh
       if:  ${{ needs.checkrights.outputs.have_secrets == 'false' || github.event_name == 'pull_request' }}
-      uses: docker/build-push-action@v2
+      uses: docker/build-push-action@v4
```

Comment:
Github warned on some old node.js usage when using v2. So updating all to v4 to be consistent.